libs/components/autonumeric/src/lib/modules/autonumeric/autonumeric.directive.spec.ts

```diff
@@ -386,20 +296,7 @@ describe('Autonumeric directive', () => {
 
   describe('Angular form control statuses', () => {
     it('should set correct statuses when initialized without value', fakeAsync(() => {
-      detectChanges();
-
-      verifyFormControlStatuses({
-        dirty: false,
-        pristine: true,
-        touched: false,
-        valid: true,
-      });
-    }));
-
-    it('should set correct statuses when initialized with falsey value', fakeAsync(() => {
```

Comment:
Is there a reason you got rid of this test?

Reply:
This test's name is a misnomer. We don't want the autonumeric directive to accept _any_ falsy value. We only want to confirm that it accepts **_numeric_** falsy values. 

Since the only numeric falsy value in JavaScript is 0, an accurate name for what the test was doing is: 

> 'should set correct statuses when initialized with **_a numeric_** falsy value'

Therefore, this test was redundant thanks to this existing test:

```
it('should properly format 0 values', fakeAsync(() => {
    detectChanges();

    setValue(0);

    detectChanges();

    const modelValue = getModelValue();
    const formattedValue = getFormattedValue();

    expect(modelValue).toEqual(0);
    expect(formattedValue).toEqual('0.00');
}));
```